contract_variable_discount/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Variable discount in contract recurrent invoicing',
+    'version': '10.0.1.2.0',
```

Comment:
'version': '10.0.1.0.0',

Reply:
@sbejaoui Please review oca documentation about version numbers here https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers
I think that the version is 10.0.1.1.0

Reply:
Hi @sergio-teruel ,

Thank you for the hint,

As this is the first release for the module, I suggested  `10.0.1.0.0` according to OCA guideline